tests/system_tests/test_main.py

```diff
@@ -229,6 +229,7 @@ def test_pre_init(self):
         out = _strip_pydevd_output(out)
         self.assertEqual(out, '')
 
+    @unittest.skipUnless(os.environ.get('HAS_NETWORK'), 'no network')
```

Comment:
I thought you were removing this.

Reply:
I thought I did too - overlooked it. I'll follow up with a PR to remove it right away (this time with legitimate VSTS test results to back the  change up!).